src/energyplus/ReverseTranslator/ReverseTranslatePerformancePrecisionTradeoffs.cpp

```diff
@@ -64,6 +64,19 @@ OptionalModelObject ReverseTranslator::translatePerformancePrecisionTradeoffs( c
     }
   }
 
+  if (optS = workspaceObject.getString(PerformancePrecisionTradeoffsFields::OverrideMode)) {
```

Comment:
Mac clang compiler threw an error on this during the build due to assignment operator in an if condition.  Easy change.

Reply:
And it's good that it did, because this is an error too :) should be `PerformancePrecisionTradeoffsFields::ZoneRadiantExchangeAlgorithm`